pythonforandroid/bootstraps/sdl2/build/src/org/kivy/android/PythonActivity.java

```diff
@@ -48,29 +48,37 @@
     private Bundle mMetaData = null;
     private PowerManager.WakeLock mWakeLock = null;
 
+    public String getKivyRoot() {
```

Comment:
Might be nice to use getAppRoot or something instead of Kivy, but it doesn't really matter.